### PR TITLE
ZENKO-3950: Add linter for node_tests

### PR DESCRIPTION
**Pull request by KillianG**

### CODE REVIEW

File: tests/zenko_tests/node_tests/backbeat/LifecycleUtility.js [hunk 1]
```diff
@@ -3,10 +3,6 @@ const async = require('async');
 const ReplicationUtility = require('./ReplicationUtility');
 
 class LifecycleUtility extends ReplicationUtility {
-    constructor(s3, azure, gcpStorage) {
-        super(s3, azure, gcpStorage);
-    }
-
```
Comment (ghost) [inline tests/zenko_tests/node_tests/backbeat/LifecycleUtility.js:9]: For my understanding: why is this not needed?

File: tests/zenko_tests/node_tests/backbeat/LifecycleUtility.js [hunk 2]
```diff
@@ -79,22 +75,22 @@ class LifecycleUtility extends ReplicationUtility {
...
+            break;
         case 'GCP':
             super.download(this.bucket, this.key, (err, data) => {
                 if (err) {
                     return cb(err);
                 }
                 // GCP returns [] for an empty object for some reason
                 if (Array.isArray(data) && data.length === 0) {
-                    return cb(null, new Buffer(0));
+                    return cb(null, Buffer.alloc(0));
```
Comment (ghost) [inline tests/zenko_tests/node_tests/backbeat/LifecycleUtility.js:86]: For my understanding: could you point me to what this does/why is this change needed?

File: tests/zenko_tests/node_tests/backbeat/tests/retry/retry.js [hunk 3]
```diff
@@ -35,78 +43,117 @@ function checkMetrics(prevBacklog, prevCompletions, prevFailures, prevPending,
...
                     next();
                 });
-            }),
+            },
+        ),
         next => awsUtils.createVersionedBucket(destFailBucket, next),
-        next => makePOSTRequest('/_/backbeat/api/crr/failed', postBody,
+        next => makePOSTRequest(
+            '/_/backbeat/api/crr/failed',
+            // eslint-disable-next-line
```
Comment (ghost) [inline tests/zenko_tests/node_tests/backbeat/tests/retry/retry.js]: For my understanding: I take the linter disable lines we see throughout are band-aid fixes so as to not make significant code logic changes that would be required by the linter while working through linter issues?

File: tests/zenko_tests/node_tests/ui/cypress/integration/bucket_spec.js [hunk 4]
```diff
@@ -9,7 +9,7 @@ describe('Bucket', () => {
 
         it('should create a bucket', () => {
             cy.createBucket(bucketName);
-            
+
```
Comment (ghost) [inline tests/zenko_tests/node_tests/ui/cypress/integration/bucket_spec.js:12]: extra space?

**Review by ghost — APPROVED** [review]
lgtm otherwise

File: tests/zenko_tests/node_tests/backbeat/LifecycleUtility.js (hunk 2 above)
Comment (KillianG) [reply tests/zenko_tests/node_tests/backbeat/LifecycleUtility.js:86]: New Buffer is deprecated, the new way to do it is with alloc, it doesnt change anything in the behaviour tho

File: tests/zenko_tests/node_tests/ui/cypress/integration/bucket_spec.js (hunk 4 above)
Comment (KillianG) [reply tests/zenko_tests/node_tests/ui/cypress/integration/bucket_spec.js:12]: I keep the line break, but I remove the useless white spaces

**Review by ghost — APPROVED** [review]
Nice!

File: tests/zenko_tests/node_tests/backbeat/tests/retry/retry.js (hunk 3 above)
Comment (KillianG) [reply tests/zenko_tests/node_tests/backbeat/tests/retry/retry.js]: You're right, will change

Comment (KillianG) [reply tests/zenko_tests/node_tests/backbeat/tests/retry/retry.js]: Changed in [a3ae3ff](https://github.com/scality/Zenko/pull/1528/commits/a3ae3ffeb90c2966771e6ba73fe4b8c485cbf4d2)

File: tests/zenko_tests/node_tests/backbeat/LifecycleUtility.js (hunk 1 above)
Comment (KillianG) [reply tests/zenko_tests/node_tests/backbeat/LifecycleUtility.js:9]: It directly calls super constructor so it is useless to keep it, as new will automatically call the super constructor
